app/_hub/kong-inc/rate-limiting/index.md

```diff
@@ -70,34 +71,37 @@ params:
     - name: second
       required: semi
       value_in_examples: 5
-      description: The amount of HTTP requests the developer can make per second. At least one limit must exist.
+      description: The number of HTTP requests the developer can make per second. At least one limit must exist.
     - name: minute
       required: semi
-      description: The amount of HTTP requests the developer can make per minute. At least one limit must exist.
+      description: The number of HTTP requests the developer can make per minute. At least one limit must exist.
     - name: hour
       required: semi
       value_in_examples: 10000
-      description: The amount of HTTP requests the developer can make per hour. At least one limit must exist.
+      description: The number of HTTP requests the developer can make per hour. At least one limit must exist.
     - name: day
       required: semi
-      description: The amount of HTTP requests the developer can make per day. At least one limit must exist.
+      description: The number of HTTP requests the developer can make per day. At least one limit must exist.
     - name: month
       required: semi
-      description: The amount of HTTP requests the developer can make per month. At least one limit must exist.
+      description: The number of HTTP requests the developer can make per month. At least one limit must exist.
     - name: year
       required: semi
-      description: The amount of HTTP requests the developer can make per year. At least one limit must exist.
+      description: The number of HTTP requests the developer can make per year. At least one limit must exist.
```

Comment:
i guess the "at least one limit must exist" refers to the whole set.  maybe it could be moved after these params list, and then enumerate them?  like "at least one of second, minute, hour, day, month, or year must specify a limit"

Reply:
Added an observation at the end.